app/celer-dump-data.cc

```diff
@@ -334,9 +334,8 @@ void print_volumes(std::vector<ImportVolume>&   volumes,
         // clang-format off
         cout << "| "
              << setw(9) << std::left << volume_id << " | "
-             << setw(11) << volume.material_id << " | "
-             << setw(36) << volume.solid_name << " | "
-             << setw(27) << material.name << " |\n";
```

Comment:
Got a `warning`: unused variable 'material'

Reply:
Restored the material output (since it makes the volume display more useful).